osis-core/src/main/java/com/scality/osis/service/impl/ScalityOsisServiceImpl.java

```diff
@@ -1137,6 +1137,16 @@ public OsisUsage getOsisUsage(Optional<String> tenantId, Optional<String> userId
         return new OsisUsage();
     }
 
+    @Override
+    public AnonymousUser getAnonymousUser() {
+        logger.info("Get Anonymous User request received");
+        AnonymousUser anonymousUser = new AnonymousUser()
+                .id(ANONYMOUS_USER_ID)
+                .name(ANONYMOUS_USER_NAME);
+        logger.info("Get Anonymous User response, {}", new Gson().toJson(anonymousUser));
```

Comment:
Is this really useful to have `info` logs heree? I feel like they can be `trace` logs, as we don't really expect anything wrong here

Reply:
agreed